src/users_db.rs

```diff
@@ -264,7 +285,14 @@ impl UsersDb {
                 email       TEXT NOT NULL UNIQUE,
                 password    TEXT NOT NULL,
                 secret      TEXT NOT NULL,
-                is_admin    BOOL NOT NULL DEFAULT 0
+                is_admin    BOOL NOT NULL DEFAULT 0,
+                push_uri    TEXT,
+                push_key    TEXT
```

Comment:
I am afraid that this schema does not scale properly.

Think about the case where we have a user signed in in multiple clients (desktop browser and mobile browser). Each client will get an independent push endpoint, but we can only store one per user.

We need separate table or even better a separate db where we can store multiple push endpoints and keys associated to a single user. And this can probably be done outside of this crate.